### PR TITLE
Radial menu atom helper

### PR DESCRIPTION
**Pull request by PsyCommando**
## Description of changes
Depends on #2303 
Added a helper to build radial menu options directly from atoms, and replaced all instances of duplicate code doing that with it.

### CODE REVIEW

File: code/_onclick/hud/radial.dm [hunk 1]
```diff
@@ -337,3 +337,19 @@ var/global/list/radial_menus = list()
 	return answer
 
 #define RADIAL_INPUT(user, choices) show_radial_menu(user, user, choices)
+
+/*
+	Helper to make a radial menu button with a name and icon for a given atom.
+*/
+/proc/make_item_radial_menu_button(var/atom/movable/AM, var/name_prefix = "", var/name_suffix = "")
+	var/image/radial_button = image(icon = AM.icon, icon_state = AM.icon_state)
```
Comment (MistakeNot4892) [inline code/_onclick/hud/radial.dm]: Possibly more rigorous to do something like:
````
var/image/radial_button = null
radial_button.appearance = AM
radial_button.plane = FLOAT_PLANE
radial_button.layer = FLOAT_LAYER
radial_button.name = etc
````

Comment (PsyCommando) [reply code/_onclick/hud/radial.dm]: Like that? 

```dm
/proc/make_item_radial_menu_button(var/atom/movable/AM, var/name_prefix = "", var/name_suffix = "")
	var/image/radial_button = image()
	radial_button.appearance = AM
	radial_button.plane = FLOAT_PLANE
	radial_button.layer = FLOAT_LAYER
	radial_button.name = "[name_prefix][AM.name][name_suffix]"
	return radial_button
```

Comment (MistakeNot4892) [reply code/_onclick/hud/radial.dm]: That looks correct.

**Comment by MistakeNot4892** [comment]
Needs a rebase now #2303 is merged.

**Review by MistakeNot4892 — APPROVED** [review]
Needs a squash on merge.

**Pull request merged**